src/Chapter1.hs

```diff
@@ -466,7 +465,7 @@ Implement a function that returns the last digit of a given number.
 
 >>> lastDigit 42
 2
-
+lastDigit x = mod x 10
```

Comment:
The implementation in here is almost correct. Unfortunately, it will fail on negative numbers because of how mod works for them. Maybe you can think of a way to fix this?


One small note, it won't work correctly on the negative numbers, unfortunately.
Here you would need to use one more standard function – to take the __abs__olute value of n 🙂

Reply:
Also, I see that you've implemented the function inside the comment, but the actual function definition is below this comment after the closing `-}` for the description. You need to move the implementation out of the comment for the compiler to check it. And don't forget to write the top-level function signature!